internal/flagv1/flag_data.go

```diff
@@ -113,7 +113,9 @@ func (f *FlagData) evaluateRule(user ffuser.User) bool {
 	}
 
 	// Evaluate the rule on the user.
-	return parser.Evaluate(f.getRule(), utils.UserToMap(user))
+	userMap := utils.UserToMap(user)
+	userMap["env"] = environment
```

Comment:
Since by default `env` is **empty** I will avoid having the key `env` in the map if there is no value.
```suggestion
	if environment != "" {
		userMap["env"] = environment
	}	
```

What do you think about that?

Reply:
makes sense to me!